src/validation.cpp

```diff
@@ -3640,7 +3654,8 @@ bool BlockManager::AcceptBlockHeader(const CBlockHeader& block, BlockValidationS
             return true;
         }
 
-        if (!CheckBlockHeader(block, state, chainparams.GetConsensus())) {
+        // During download headers, do not CheckBlockHeader for performance reason // See https://github.com/sugarchain-project/sugarchain/pull/122
+        if (!::ChainstateActive().IsInitialBlockDownload() && !CheckBlockHeader(block, state, chainparams.GetConsensus())) {
```

Comment:
assume this made fuzzing failure
https://travis-ci.org/github/sugarchain-project/yumekawa/builds/740341268#L3642